ginivision/src/main/java/net/gini/android/vision/review/ReviewActivity.java

```diff
@@ -321,6 +321,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 setResult(resultCode, data);
```

Comment:
I know that `mBackButtonShouldCloseLibrary` confuses things. We used that when we changed the behavior to allow clients to revert to the old behavior. I'll create an issue to flag it deprecated and remove it in a few months.